services/QuillLMS/spec/services/vitally_integration/serialize_vitally_sales_account_spec.rb

```diff
@@ -276,6 +276,24 @@
 
       it { expect(results[:traits][:active_students]).to eq(2) }
     end
+
+    context 'when teachers are merely coteachers and not owners of the classroom' do
+      it 'does not double count activities finished and active students' do
+        classroom_teachers[0].update(role: 'coteacher')
+        classroom_teachers[1].update(role: 'coteacher')
```

Comment:
Take or leave this, but I'd probably do the coteacher change in a `let` call inside the context:
```
let!(:classroom_teachers) { create_list(:classrooms_teacher, 2, classroom:, role: 'coteacher')

it 'does not double count activities finished and active students' do
  ...
```
Just to keep the `it` block super simple.